axes/helpers.py

```diff
@@ -148,6 +148,24 @@ def get_client_username(
     return request_data.get(settings.AXES_USERNAME_FORM_FIELD, None)
 
 
+def get_client_ip_address_default(request: HttpRequest) -> str:
+    try:
+        import ipware.ip
+    except ImportError as e:
+        raise AxesBackendImplementationRequired(
+            "Axes requires either configuring settings.AXES_CLIENT_IP_CALLABLE django-ipware package to be installed with e.g. pip install django-axes[ipware]."
+        ) from e
```

Comment:
This try/except block will cause python to try to import `ipware` during each function call. Instead, you can do smth like that in head of the module:
```python
try:
    import ipware.ip
    IPWARE_INSTALLED = True
except ImportError:
     IPWARE_INSTALLED = False
```
and in function body:
```python
if not IPWARE_INSTALLED:
    raise ...

Reply:
Yeah, let's do that 👍